sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -62,6 +64,46 @@ def test_generate_policy():
         assert len([t for t in topics if t.text == 'topic_pub']) == 0
 
 
+def test_generate_policy_services():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        # Create a test-specific context so that generate_policy can still init
+        context = rclpy.Context()
+        rclpy.init(context=context)
+        node = rclpy.create_node('test_node', context=context)
+
+        try:
+            # Create a server and client
+            node.create_client(GetParameters, 'service_client')
+            node.create_service(GetParameters, 'service_server', lambda request,
```

Comment:
Since nodes already create a [`GetParameters` service](https://github.com/ros2/rclpy/blob/72f55462e7fb9c0011904da9949557996ba56396/rclpy/rclpy/parameter_service.py#L38-L39) as part of the `ParameterServer`, I suggest using a different service type, maybe from [std_srvs](https://github.com/ros2/common_interfaces/tree/master/std_srvs/srv).